helm-charts/0.21.6/charts/safe/templates/identity.yaml

```diff
@@ -13,7 +13,7 @@ kind: ClusterSPIFFEID
 metadata:
   name: {{ include "safe.fullname" . }}
 spec:
-  spiffeIDTemplate: "spiffe://vsecm.com\
+  spiffeIDTemplate: "spiffe://{{ .Values.global.spire.trustDomain }}\
```

Comment:
The customer might want their trust domain (for i.e. theBigTelcoCompany.com) instead of `vsecm.com` 

This provides the flexibility.